docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-list/index.scss

```diff
@@ -0,0 +1,36 @@
+@use 'uswds-core' as *;
+@forward 'usa-collection';
+@import "~Styles/sprites/svg-sprite";
+
+.usa-collection__heading {
+	.usa-link {
+		@include u-font('serif', 'md');
+	}
+}
+
+// TODO: Remove once DS has been updated
+// This is a fix to remove spacing on a collection item
+// when there is no list title.
+.usa-section {
+  .cgdp-list:not(:has(> h2)) {
+		.usa-collection {
+			@include u-margin-top(0);
+			.usa-collection__item:first-child {
+				@include u-padding-top(0);
+			}
+		}
+	}
+}
+
+.usa-collection__img {
+	@include u-margin-bottom(1.5);
+	vertical-align: middle;
+}
```

Comment:
You should scope these to cgdp-list AND make the comment clearer what should be updated. The NCIDS ticket number should be included in the ticket. If you don't have it, I guess you will need to wait.